configure.ac

```diff
@@ -1394,7 +1433,7 @@
 
         AC_ARG_WITH(libhtp_includes,
                 [  --with-libhtp-includes=DIR  libhtp include directory],
-                [with_libhtp_includes="$withval"],[with_libhtp_includes=no])
+                [with_libhtp_includes="$withval"],[with_libhtp_includes="no"])
```

Comment:
Please make this a separate commit as it's unrelated to windivert

Reply:
OK.